heroku3/helpers.py

```diff
@@ -30,15 +33,16 @@ def is_collection(obj):
 
 # from kennethreitz/python-github3
 def to_python(obj,
-    in_dict,
-    str_keys=None,
-    date_keys=None,
-    int_keys=None,
-    object_map=None,
-    array_map=None,
-    bool_keys=None,
-    dict_keys=None,
-    **kwargs):
+              in_dict,
+              strs=None,
+              dates=None,
+              ints=None,
+              objects=None,
+              arrays=None,
+              bools=None,
+              dicts=None,
```

Comment:
Made the arguments names more similar to those of class attributes used to declare the models attributes. Makes testing a bit easier.